packages/backend/src/tokens/authObjects.ts

```diff
@@ -34,6 +34,13 @@ export type SignedInAuthObject = {
   orgRole: OrganizationCustomRoleKey | undefined;
   orgSlug: string | undefined;
   orgPermissions: OrganizationCustomPermissionKey[] | undefined;
+  /**
+   * Factor Verification Age
+   * Each item represents the minutes that have passed since the last time a first or second factor were verified.
+   * [fistFactorAge, secondFactorAge]
+   * @experimental This API is experimental and may change at any moment.
+   */
+  __experimental_fva: [number | null, number | null];
```

Comment:
Gut feeling: This should be called `__experimental_factorVerificationAge` for public consumption everywhere, too. We're not really doing pure acronyms in other places and I wondered why we use another name internally

Reply:
We are not expecting this to be consumed directly, people should be using `protect | has({ assurance: {..} })` which abstracts fva away.

We only need to expose and provide the fva value to ClerkProvider internally in order to ensure SSR working properly.

Reply:
@chanioxaris Do you think we should rename ?